### ISSUE TITLE
MySQL metadata queries pile up and run forever (until killed).

### ISSUE THREAD

**Issue by stefanproell** [metabase/metabase#7112]
Metabase starts new metadata queries periodically, regardless of the state of the previous query and produces a high load.

One of our databases in a MySQL 5.6 instance contains a computationally intensive view use for some internal data collection, that queries a large table and therefore requires multiple hours to complete. In the process list we see ~100 queries on the same view, an example is below:

```
Command: Query
   Time: 2816
  State: Sending data
   Info: -- Metabase
SELECT count(*) AS `count` FROM `intensive_view`
```
We have to kill these queries manually in order to calm down the system. A timeout for this kind of metadata queries or a check if the subsequent check is still running would be beneficial. As mitigation, I removed the view form the metabase catalog to a different database. 

Thanks,
kind regards
Stefan

**Comment by senior** [comment]
This issue should be resolved as of 0.28.0 with https://github.com/metabase/metabase/pull/6612. That will ensure that the sync processes will not "pile up" and end up running concurrently. Can you upgrade and verify the fix?

The specific query you're mentioning is the row count queries which have been removed from the sync process entirely with https://github.com/metabase/metabase/pull/6612. That will be included in version 0.29.0 which will be out soon.

**Comment by senior** [comment]
Closing this as it's fixed in master and will be released soon.
